packages/patterns/src/components/create-pattern-modal.js

```diff
@@ -36,16 +36,18 @@ export default function CreatePatternModal( {
 	const [ syncType, setSyncType ] = useState( PATTERN_SYNC_TYPES.full );
 	const [ categories, setCategories ] = useState( [] );
 	const [ title, setTitle ] = useState( '' );
+	const [ categorySaving, setCategorySaving ] = useState();
```

Comment:
Nit: I would probably name it `saveCategoryPromise` to make it clear that the value is a promise instance.

Reply:
Yeh, that is clearer, thanks, have done.